src/main/resources/db/changelog/changeset/02-create-users-table.yaml

```
@@ -0,0 +1,39 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-table
+      author: Vladyslav_Vyshynskyi
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: email
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: firstname

```

same for other fields
```suggestion
                  name: first_name

```